docs/src/main/asciidoc/command-mode-reference.adoc

```diff
@@ -20,25 +20,28 @@ that exit.
 In this document the `QuarkusApplication` instance is referred to as the application main,
 and a class with a Java main method is the Java main.
 
-The simplest possible command mode application might appear as follows:
+The simplest possible command mode application with access to Quarkus API's might appear as follows:
 
 [source,java]
 ----
 import io.quarkus.runtime.QuarkusApplication;
 import io.quarkus.runtime.annotations.QuarkusMain;
 
-@QuarkusMain    // <1>
+@QuarkusMain    // <.>
 public class HelloWorldMain implements QuarkusApplication {
   @Override
-  public int run(String... args) throws Exception {   // <2>
+  @ActivateRequestContext // <.>
+  public int run(String... args) throws Exception {   // <.>
     System.out.println("Hello World");
     return 10;
  }
 }
 ----
-<1> The `@QuarkusMain` annotation tells Quarkus that this is the main entry point.
-<2> The `run` method is invoked once Quarkus starts, and the application stops when it finishes.
+<.> The `@QuarkusMain` annotation tells Quarkus that this is the main entry point.
+<.> The `@ActivateRequestContext` annotation is not strictly required for this example, but ensures you have access to context dependency injection. Allowing you to use API's such as Hibernate Panache.
```

Comment:
As discussed on zulip, this sentence is wrong. `@ActivateRequestContext` merely activates the request context and makes it possible to use `@RequestScoped` beans in the command mode.

Reply:
@mkouba what would be the cost of activating it always?

Reply:
> what would be the cost

Negligible. So in general, a command execution is a kind of a request. The only concern I have is that someone can use the command mode to execute a relatively long-lived app and since the automatic activation turns `@RequestScoped` into `@ApplicationScoped` it loses its meaning...

Reply:
I'm going to update this pr - cli's can easily be long lived and have multiple (even concurrent) requests thus it makes sense to be explicit wehn you want one over the other.